lib/packages/docutils/rst.nim

```diff
@@ -840,7 +840,8 @@ proc isInlineMarkupEnd(p: RstParser, markup: string): bool =
   if not result: return
   # Rule 4:
   if p.idx > 0:
-    if markup != "``" and prevTok(p).symbol == "\\":
+    # refs bug #17260
+    if markup notin ["``", "`"] and prevTok(p).symbol == "\\":
```

Comment:
Please, no. You are committing a double sin here: spec noncompliance and inability to input `` ` ``. Again, case of a standalone `\` can be covered by
```
``\``
```

Reply:
fixed